polkadot/node/network/protocol/src/request_response/mod.rs

```diff
@@ -38,7 +38,7 @@
 //! `fallback_name` to the protocol config.
 //!
 //! One way in which versioning has historically been achieved for req-response protocols is to
-//! bundle the new req-resp version with an upgrade of a notifications protocol. The subsystem would
+//! bundle the new req-resp version with an upgrade of a notifications' protocol. The subsystem would
```

Comment:
```suggestion
//! bundle the new req-resp version with an upgrade of a notifications protocol. The subsystem would
```

I believe here the term "notifications protocol" is being used as its own thing, and not as the protocol of the notifications.

Reply:
Got it!